syntax = "proto3";
package demo;

service Chat {
  rpc Subscribe(SubscribeRequest) returns (stream ChatMessage);
}

message SubscribeRequest {
  string client_id = 1;
}

message ChatMessage {
  string text = 1;
}
